kyuubi-common/src/main/scala/org/apache/kyuubi/service/TFrontendService.scala

```diff
@@ -532,9 +533,27 @@ abstract class TFrontendService(name: String)
   }
 
   override def SetClientInfo(req: TSetClientInfoReq): TSetClientInfoResp = {
+    // TODO: We don't do anything for now, just log this for debugging.
```

Comment:
Why do you add the `TODO` here? Any update in future?

Reply:
@yangrong688, any feedback?

Reply:
Thanks for your kindly comments. Based on Hive master branch, there is no further update here, I'll remove TODO here and update later once Hive have some changes.